docs/1.getting-started/3.views.md

```diff
@@ -129,6 +129,16 @@ If you only have a single layout in your application, we recommend using app.vue
 </template>
 ```
 
+```vue [pages/app.vue]
```

Comment:
Firstly, this would create a `/app` page, not an app entrypoint. And secondly, the library already provided a `App.vue` file at https://github.com/nuxt/nuxt/blob/main/packages/nuxt/src/pages/runtime/app.vue so it's not necessary to create a `app.vue` file yourself to get started with Nuxt.